jsonnet/components/node-exporter.libsonnet

```diff
@@ -120,18 +121,8 @@ function(params)
     // configured on the `Service` above and adds the default init text
     // collectors to the process.
     daemonset+: {
-      metadata+: {
-        labels+: {
-          'app.kubernetes.io/managed-by': 'cluster-monitoring-operator',
-        },
-      },
       spec+: {
         template+: {
-          metadata+: {
-            labels+: {
-              'app.kubernetes.io/managed-by': 'cluster-monitoring-operator',
```

Comment:
we make upstream inject it now (even though CMO doesn't manage the Pods directly))